providers/logrus/provider.go

```diff
@@ -9,9 +9,9 @@ package logrus
 import (
 	"github.com/sirupsen/logrus"
 
-	"github.com/myhelix/contextlogger/log"
-	"github.com/myhelix/contextlogger/providers"
-	"github.com/myhelix/contextlogger/providers/chaining"
+	"github.com/myhelix/contextlogger/v2/log"
+	"github.com/myhelix/contextlogger/v2/providers"
+	"github.com/myhelix/contextlogger/v2/providers/chaining"
```

Comment:
How does this work? this is what I'd expect from the "maintain v2 in a directory" option, not the branch option.